rqt_py_common/CMakeLists.txt

```diff
@@ -1,26 +1,33 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(rqt_py_common)
-# Load catkin and all dependencies required for this package
-find_package(catkin REQUIRED)
-catkin_python_setup()
 
-if(CATKIN_ENABLE_TESTING)
-  find_package(genmsg REQUIRED)
-  find_package(std_msgs REQUIRED)
-  add_message_files(
-    DIRECTORY test/msg
-    FILES ArrayVal.msg Val.msg
-    NOINSTALL
-    )
-  generate_messages(DEPENDENCIES std_msgs)
-endif()
+# Load catkin and all dependencies required for this package
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_python REQUIRED)
 
-catkin_package()
+ament_python_install_package(${PROJECT_NAME}
+  PACKAGE_DIR src/${PROJECT_NAME})
 
 install(DIRECTORY resource
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+  DESTINATION share/${PROJECT_NAME}
 )
 
-if(CATKIN_ENABLE_TESTING)
-  catkin_add_nosetests(test)
-endif()
\ No newline at end of file
+install(TARGETS ${_library_name}
```

Comment:
Where is the variable `_library_name` being defined?

Reply:
Resolved in #155